src/sdl/sdl_mouse.c

```diff
@@ -14,6 +14,15 @@ typedef struct ALLEGRO_MOUSE_SDL
 static ALLEGRO_MOUSE_DRIVER *vt;
 static ALLEGRO_MOUSE_SDL *mouse;
 
+static float sdl_get_display_pixel_ratio(ALLEGRO_DISPLAY *display)
```

Comment:
Maybe make this internal (i.e. put a declaration in include/allegro5/platform/allegro_internal_sdl.h) ? You use this in a few more place than just in this file.